suite-common/redux-utils/src/extraDependenciesType.ts

```diff
@@ -11,25 +12,36 @@ type StorageLoadReducer = (state: any, action: { type: any; payload: any }) => v
 
 export type ExtraDependencies = {
     thunks: {
-        fetchAndUpdateAccount: SuiteCompatibleThunk<Account>;
         notificationsAddEvent: SuiteCompatibleThunk<NotificationEventPayload>;
     };
     selectors: {
         selectFeeInfo: (networkSymbol: NetworkSymbol) => SuiteCompatibleSelector<FeeInfo>;
-        selectAccounts: SuiteCompatibleSelector<Account[]>;
         selectDevices: SuiteCompatibleSelector<TrezorDevice[]>;
         selectBitcoinAmountUnit: SuiteCompatibleSelector<PROTO.AmountUnit>;
     };
     // You should only use ActionCreatorWithPayload from redux-toolkit!
     // That means you will need to convert actual action creators in packages/suite to use createAction from redux-toolkit,
     // but that shouldn't be problem.
-    actions: {};
+    actions: {
+        // FIXME: these types for actions are temporary - remove it when Bohdan is ready with wallet-transactions!
+        addTransaction: (
```

Comment:
Please always use utility types `ActionCreatorWithPayload` from redux-toolkit. I know it will throw error because it won't work with normal actions, but that is on purpose to force you to migrate these actions to `createAction`. I know it only very temporary because Bohdan will remove it soon anyway, but I don't want to give any space to set bad examples. And it's super easy to migrate it anyway:

```tsx
export const add = (
    transactions: AccountTransaction[],
    account: Account,
    page?: number,
): TransactionAction => ({
    type: TRANSACTION.ADD,
    transactions: transactions.map(tx => enhanceTransaction(tx, account)),
    account,
    page,
});
// will become
const add = createAction(
    TRANSACTION.ADD,
    (transactions: AccountTransaction[], account: Account, page?: number) => ({
        payload: {
            transactions: transactions.map(tx => enhanceTransaction(tx, account)),
            account,
            page,
        },
    }),
);
```